suites/nautilus/live/live_sanity_container.yaml

```diff
@@ -0,0 +1,103 @@
+tests:
+- test:
+    name: install ceph pre-requisites
+    module: install_prereq.py
+    abort-on-fail: True
+
+- test:
+    name: ceph ansible install containerized rhcs 4.x from cdn
+    polarion-id: CEPH-83573588
+    module: test_ansible.py
+    config:
+      use_cdn: True
+      build: '4.x'
```

Comment:
Do we need the key `build`? I don't to have a reason to indicate it is not required but it is kind of misleading when we say `4.x`...

Reply:
ok will remove that config option key **build**, as i was using earlier yaml hence it got retained.
Thanks